voluncheer/voluncheer/urls.py

```diff
@@ -18,28 +18,14 @@
 from django.urls import path
 from django.contrib.auth import views as auth_views
 
-from profiles.views.home import SignUpView
-from profiles.views.organizations import OrganizationSignUpView
-from profiles.views.volunteers import VolunteerSignUpView
 
 urlpatterns = [
-    path("__debug__/", include("debug_toolbar.urls")),
-    path("admin/", admin.site.urls),
+    path("", include("django.contrib.auth.urls")),
     path("", include("profiles.urls")),
-    path("accounts/", include("django.contrib.auth.urls")),
-    path("accounts/signup/", SignUpView.as_view(), name="signup"),
-    path(
-        "accounts/signup/organization/",
-        OrganizationSignUpView.as_view(),
-        name="organization_signup",
-    ),
-    path(
-        "accounts/signup/volunteer/",
-        VolunteerSignUpView.as_view(),
-        name="volunteer_signup",
-    ),
+    path("admin/", admin.site.urls),
```

Comment:
Odd that this one isn't in an `include()` call. Would you happen to know why that is?

Reply:
For whatever reason, it's not necessary. See: https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#hooking-adminsite-to-urlconf

Reply:
/shrug I guess we have to just "take Django's word for it".